### PR TITLE
How to db

### PR DESCRIPTION
**Pull request by wcpines**

### CODE REVIEW

**Comment by gib** [comment]
Do you even db bro?

File: source/topics/cli/how-to-dump-and-restore.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+If you want to migrate data from one Aptible database to another, or perform a back-up and restore, you can use the Aptible CLI with the following steps:
```
Comment (fancyremarker) [inline source/topics/cli/how-to-dump-and-restore.md]: Just "backup", not "back-up."

File: source/topics/cli/how-to-dump-and-restore.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    aptible db:tunnel $ORIGIN_DB_HANDLE
+    ```
+
+2. Launch a database client and use the relevant appropriate command to dump your data locally, e.g.
+
+    ```bash
+    mysqldump -u aptible -p[PASSWORD] -h 127.0.0.1 -P [PORT] db > dumpfilename.sql
+    ```
+
+    If you're using postgres, you can combine steps 1 and 2 with `aptible db:dump $HANDLE`
```
Comment (fancyremarker) [inline source/topics/cli/how-to-dump-and-restore.md]: Let's use "PostgreSQL" for consistency.

File: source/topics/cli/how-to-dump-and-restore.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+3.  Provision a new Aptible database if you have not already done so.
+
+4. Similar to the first two steps, connect to the new database and, in a new bash session, upload the dumped file:
+
+    ```bash
+    aptible db:tunnel $NEW_DB_HANDLE
+
+    # separate session
+    mysqldump -u aptible -p[PASSWORD] -h 127.0.0.1 -P [PORT] db < dumpfilename.sql
```
Comment (fancyremarker) [inline source/topics/cli/how-to-dump-and-restore.md]: This shouldn't be `mysqldump`, but `mysql`.

**Comment by wcpines** [comment]
Corrections made

**Pull request merged**